functest/send_money_test.go

```diff
@@ -128,11 +132,11 @@ func TestTransferMoreThanAvailableAmount(t *testing.T) {
 	oldFirstBalance := getBalanceNoErr(t, firstMember, firstMember.ref)
 	oldSecondBalance := getBalanceNoErr(t, secondMember, secondMember.ref)
 
-	amount := oldFirstBalance + 100
-
-	_, err := signedRequest(firstMember, "Transfer", amount, secondMember.ref)
-	require.Contains(t, err.Error(), "[ Transfer ] Not enough balance for transfer: subtrahend must be smaller than minuend")
+	amount := new(big.Int)
+	amount.Add(oldFirstBalance, big.NewInt(10))
 
+	_, err := signedRequest(firstMember, "wallet.transfer", map[string]interface{}{"amount": amount.String(), "to": secondMember.ref})
+	require.NotNil(t, err)
```

Comment:
Its better to check error message here also, to be sure its about more than available amount, not smth else